copyq/__init__.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+
+"""Access CopyQ clipboard.
+
```

Comment:
Docsting is intended for exhaustive descritpion. DRY. There are functions for brief desc and synopsis. Rather leave it empty